### PR TITLE
wip: v4 aries lbw

### PR DESCRIPTION
**Pull request by bschwal**
Adding aries latency and bandwidth sampler under the aries_mmr sampler folder.

### CODE REVIEW

**Comment by bschwal** [comment]
@oceandlr I will add that to the man page, should I have a retoggle perms if the first try fails in my sample as well?

**Comment by bschwal** [comment]
The changes have been made in the latest commit. I tested it on the whole of ldms2 partition on mutrino for application runs and it is sampling and storing without error.

File: ldms/src/sampler/aries_mmr/aries_lbw.c [hunk 1]
```diff
@@ -0,0 +1,598 @@
+/* -*- c-basic-offset: 8 -*-
+ * Copyright (c) 2015-2017 National Technology & Engineering Solutions
```
Comment (narategithub) [inline ldms/src/sampler/aries_mmr/aries_lbw.c]: Shouldn't the year be 2020?

File: ldms/src/sampler/aries_mmr/aries_lbw.c [hunk 2]
```diff
@@ -0,0 +1,598 @@
...
+ * of Sandia, LLC (NTESS). Under the terms of Contract DE-NA0003525 with
+ * NTESS, the U.S. Government retains certain rights in this software.
+ * Copyright (c) 2015-2017 Open Grid Computing, Inc. All rights reserved.
```
Comment (narategithub) [inline ldms/src/sampler/aries_mmr/aries_lbw.c]: I think OGC does not have a copyright to this code. @tom95858 could you please verify?

File: ldms/src/sampler/aries_mmr/aries_lbw.c (hunk 1 above)
Comment (bschwal) [reply ldms/src/sampler/aries_mmr/aries_lbw.c]: I'm seeing a lot of samplers that are 2017/2018 in OVIS-4, perhaps this needs a refresh for all?

File: ldms/src/sampler/aries_mmr/aries_lbw.c (hunk 2 above)
Comment (bschwal) [reply ldms/src/sampler/aries_mmr/aries_lbw.c]: Same as above, lots of samplers have this.

Comment (tom95858) [reply ldms/src/sampler/aries_mmr/aries_lbw.c]: correct, not to my knowledge

**Comment by tom95858** [comment]
You should not just arbitrarily advance the end date of a copyright without modifying the underlying source code. Is there some background to this I should be aware of?

**Comment by bschwal** [comment]
I was simply copying from aries_mmr source code as the basis for this one. Completely fine with changing this copyright as appropriate.

**Comment by tom95858** [comment]
@bschwal what is the status of this pull request? Does it need to be closed?

**Comment by tom95858** [comment]
Please re-open if it is ready for OVIS-4.

**Pull request closed without merging**